### PR TITLE
Fix input component styling for dark mode

### PR DESCRIPTION
**Pull request by fgonza23**
Updated: 
Problem
Initially, the issue was identified as the text not rendering correctly in dark mode. However, the root cause was that the background of certain elements wasn’t properly adapting to the dark mode styles.

Solution
Applied the dark:bg class to the input and textarea components to ensure their backgrounds render correctly in dark mode.
Removed redundant or conflicting classes from label elements to maintain consistent styling.

Updates
Removed .index.tsx.swp from Git's history.
Added a rule to .gitignore to ignore all .swp files, preventing them from being tracked in the future.

### CODE REVIEW

**Review by jvmontes — DISMISSED** [review]
After a call with Francisco, just gotta remove the unnecessary text color setting and this should be good to go.

**Review by rogeruiz — CHANGES_REQUESTED** [review]
❌ This PR needs changes ...

Looks good but the `.swp` file needs to be removed @fgonza23 

```sh
# remove the file from Git's history
git rm --cached src/components/NavBar/MenuButton/.index.tsx.swp

# add `.swp` files to the local `.gitignore`.
echo "*.swp" >> .gitignore

# update the project's ignorance towards `.swp` files
git add .gitignore
```

I'd also like to see all `.swp` files add to the project's `.gitignore` to prevent it from happening again in the future. I've included the steps above. But hit me up if you have any questions.

File: src/components/common/Input/index.tsx [hunk 1]
```diff
@@ -13,15 +13,18 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
         {...props}
         id={id}
         ref={ref}
-        className="h-12 rounded-md border-2 border-gray-300 pl-2"
+        className="h-12 rounded-md border-2 border-gray-300 pl-2 text-black dark:bg-gray-800 dark:text-white"
```
Comment (jvmontes) [inline src/components/common/Input/index.tsx:16]: Do we still need this here?

File: src/components/common/Input/index.tsx [hunk 2]
```diff
@@ -13,15 +13,18 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
...
     );
 
     if (!label) return textField;
 
     return (
       <div className={`flex flex-col ${className}`}>
-        <label htmlFor={id} className="mb-2 font-bold">
+        <label
+          htmlFor={id}
+          className="mb-2 font-bold text-black dark:text-white"
```
Comment (jvmontes) [inline src/components/common/Input/index.tsx:26]: Do we still need this text-black here?

**Review by jvmontes — CHANGES_REQUESTED** [review]

**Review by jvmontes — APPROVED** [review]
nice & clean ✨

**Pull request merged**